index.js

```diff
@@ -39,6 +39,11 @@ const make = (files, opts) => {
 			continue;
 		}
 
+		if (['vim', 'neovim'].indexOf(editor.id) !== -1) {
+			args.push(`"+call cursor(${parsed.line}, ${parsed.column})"`, parsed.file);
```

Comment:
Does it really have to be wrapped in an inner `"`? Arguments are already quoted by Node.js. Can you test if it works without?